pkg/util/proxy.go

```diff
@@ -39,7 +39,10 @@ type ProxyConfig struct {
 // the corresponding environment variable is checked.
 func NewProxyConfig(httpProxy string, httpsProxy string, noProxy string) (*ProxyConfig, error) {
 	if httpProxy == "" {
-		httpProxy = os.Getenv("HTTP_PROXY")
+		httpProxy = os.Getenv("http_proxy")
```

Comment:
This is technically out of scope but I hope that we can still get this in here: Some tools accept a lower case `http_proxy` env var as well an upper case `HTTP_PROXY` and friends. In curl, lower case env vars take precedence over upper case ones.

Reply:
Please address as a separate issue